JoinColony/colonyCDapp
# feat: add search pill to filters

## Description

- Added search pill next to filters
- When clicking 'enter' key while typing in search input - filter menu closes
<img width="1235" alt="image" src="https://github.com/user-attachments/assets/749e6654-d857-4956-9b6b-b107293f6607">

## Testing

* Go to any page with filters i.e. activity feed, balances
* Open filters menu and use search
* Click 'enter' key and check if menu closes

Resolves https://github.com/JoinColony/colonyCDapp/issues/3666
